packages/nodes-base/credentials/CiscoUmbrellaApi.credentials.ts

```diff
@@ -27,7 +27,7 @@ export class CiscoUmbrellaApi implements ICredentialType {
 		{
 			displayName: 'API Key',
 			name: 'apiKey',
-			// eslint-disable-next-line n8n-nodes-base/cred-class-field-unobscured-sensitive-input
+			// eslint-disable-next-line n8n-nodes-base/cred-class-field-type-options-password-missing
```

Comment:
Kept this as it was, but why are we exempting this?

Reply:
Good question I am not sure, I would say API keys should be impacted as well.